### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Andrey1104**

### CODE REVIEW

File: app/customer.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    def get_money(self) -> None:
+        print(f"{self.name} has {self.money} dollars")
+
+    def get_distance(self, other: Shop) -> float:
+        return (
+            round((sqrt((other.location[0]
+                         - self.location[0]) ** 2
+                        + (other.location[1]
+                           - self.location[1]) ** 2)) * 2, 2)
+        )
```
Comment (viktoria-rybenchuk) [inline app/customer.py]: consider breaking down the expression into smaller, more descriptive variables to enhance clarity.

File: app/user_data.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+path = os.path.abspath(os.path.join(os.path.dirname(__file__), "config.json"))
+
+
+def get_data() -> dict:
+    with open(path, "r") as text:
+        data = json.load(text)
+    return data
+
+
+FUEL_PRICE = float(get_data()["FUEL_PRICE"])
```
Comment (viktoria-rybenchuk) [inline app/user_data.py]: Is `FUEL_PRICE` related to  user data?

File: app/main.py [hunk 3]
```diff
@@ -1,3 +1,48 @@
...
+
+
+def shop_trip() -> None:
+    customers = [
+        Customer(*customer.values())
+        for customer in get_data()["customers"]
+    ]
+    shops = [Shop(*shop.values()) for shop in get_data()["shops"]]
+    date = datetime.datetime.now()
+    date = date.strftime("%d/%m/%Y %H:%M:%S")
```
Comment (viktoria-rybenchuk) [inline app/main.py]: write it in one variable

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

File: app/user_data.py (hunk 2 above)
Comment (Andrey1104) [reply app/user_data.py]: no, therefore it is used in the method as an imported constant. is it prolblem?

Comment (viktoria-rybenchuk) [reply app/user_data.py]: It's important to keep  unrelated data in a separate file in order to enhance modularity and maintainability

**Review by viktoria-rybenchuk — APPROVED** [review]
Good job!

**Pull request closed without merging**